modules/export_pipeline/spatialize_chunk_predictions.tf

```diff
@@ -93,8 +93,9 @@ resource "google_cloudfunctions2_function" "spatialize_chunk_predictions_functio
   }
 
   service_config {
-    available_memory    = "4Gi"
+    available_memory    = "32Gi"
     timeout_seconds    = 540
+    max_instance_count = 3000 // (~600 chunks x 4 scenarios)
```

Comment:
I believe the default max is much lower than 3K, which means we can't process too many chunks in parallel

Reply:
oh, sorry, I misread that as min, which should be 0
from the docs:

> The default [maximum instances limit](https://cloud.google.com/functions/docs/configuring/max-instances) for 2nd gen HTTP functions is 100 and can be increased to 1,000.

https://cloud.google.com/functions/quotas#:~:text=The%20default%20maximum%20instances%20limit,for%201st%20gen%20HTTP%20functions.

I doubt we actually have quota for 1,000, though.
I wouldn't worry too much about total processing time for this.

Reply:
So, let's not set this above 1,000 as that's the actual max

Reply:
This is an event-driven function so the max_instance_count is higher. Even with a max instance count of 3000, not all chunks would run in parallel (since there are a total of 8 scenarios). 

I'm OK to set this lower but out of curiosity what would be the benefit? I don't see pricing being affected by using more instances: https://cloud.google.com/functions/pricing

Reply:
Updated to 1000 as discussed offline